### ISSUE TITLE
SQL-Query task frequency.

### ISSUE THREAD

**Issue by hayriozler** [opserver/Opserver#383]
Hi,

Is it possible to create different frequency and configuration screen foreach SQL queries. for an example

I would like to run : CPU utilization query every 5 seconds but Database but top expensive queries every 10 seconds etc.

**Comment by JeevaDev7** [comment]
Yes, it is possible to create different frequencies and configurations for running specific SQL queries. The implementation depends on the tools and frameworks you are using to execute and schedule these queries.
